content/blog/debug-cran-additional.md

```diff
@@ -0,0 +1,384 @@
+---
+title: "Debugging and Fixing CRAN's 'Additional Checks' errors"
+date: 2020-01-16
+tags:
+ - dde
+ - R
+---
+
+R packages that are published on CRAN are tested every night on a variety of platforms and on the development version of R to ensure that they continue to work.  In addition, packages that contain compiled code (C, C++ or Fortran) are put through a raft of additional checks to ensure that the compiled code will not cause R to crash.  Once an issue is found, the package maintainer gets an email and usually a fairly short window to fix the package before it is removed from CRAN.  However, replicating the error locally can require installation of all sorts of esoteric tools (and a copy of the development version of R from source) and it's not always obvious to start.
+
```

Comment:
possibly colloquialism - did you mean "where" to start or "how" to start - or perhaps it's fine as it is.